### PR TITLE
Update firefox PPA

### PR DESCRIPTION
**Pull request by lmacka**
I haven't tested this in a while.  There's a few readme updates you might like to filter out.

### CODE REVIEW

**Comment by Jeff-liaojinfu** [comment]
您的邮件我已经收到，我会尽快回复您！

File: Dockerfile [hunk 1]
```diff
@@ -2,9 +2,11 @@ FROM ubuntu:14.04
 MAINTAINER Kyle Anderson <kyle@xkyle.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y install software-properties-common
+RUN add-apt-repository ppa:forkotov02/firefox-gtk2-trusty
 RUN apt-get update && apt-get -y install xvfb x11vnc wget \
-    supervisor fluxbox icedtea-7-plugin net-tools python-numpy \
-    chromium-browser
```
Comment (solarkennedy) [inline Dockerfile:7]: Can't we have chromium && firefox?

File: Dockerfile [hunk 2]
```diff
@@ -2,9 +2,11 @@ FROM ubuntu:14.04
 MAINTAINER Kyle Anderson <kyle@xkyle.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y install software-properties-common
```
Comment (solarkennedy) [inline Dockerfile:5]: why do we need this PPA at all? Is the version of firefox in ubuntu 14.04 not good enough?

**Comment by solarkennedy** [comment]
This PR isnt really mergeable.
Please make a PR with just the firefox change.

**Pull request closed without merging**